bin/lint_trailing_spaces.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Report any .toml files that have trailing white space
+# so user can fix them
+set -eo pipefail
+
+files=$(find . -name '*.toml' -exec grep -l '[[:space:]]$' {} \;)
```

Comment:
```suggestion
files=$(find . -iname '*.toml' -o -iname '*.rs' -o -iname '*.md' -exec grep -l '[[:space:]]$' {} \;)
```

`-name` -> `-iname` just makes the match case-insensitive, in case someone ever creates `IMPORTANT.MD` or something.

I think that we want to lint rust and markdown files as well.

Reply:
Awesome! So our current markdown lint will cover `.md` files so perhaps we only do `.rs` and `.sh`?

Reply:
Sure, works for me.